spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe "the guest show page" do
+  before :each do
+    @hotel1 = Hotel.create!(name: 'Echo Mountain Inn', location: 'Echo Mountain')
+    @hotel2 = Hotel.create!(name: 'Reverb Hill Lodge', location: 'Reverb Hill')
+    @room1 = Room.create!(rate: 125, suite: "Presidential", hotel_id: @hotel1.id)
```

Comment:
Other options for making rooms:
`@hotel1.rooms.create!((rate: 125, suite: "Presidential")` or `Room.create!(rate: 125, suite: "Presidential", hotel: @hotel1)`
What you did is perfectly fine, just want to remind you of the other options! :)